lib/index.js

```diff
@@ -353,6 +354,71 @@ const feedbackHandler = async function feedbackHandler(options) {
   logger.info('Thank you!');
 };
 
+async function triggerUpsertHandler(options) {
+  // We have to use the position number here because of lack of good support
+  // for nested commands with positional arguments
+  // consider replacing with `nested-yargs` npm package
+  const triggerName = options._[2];
+  const triggerOptions = {
+    source: {
+      uri: options.sourceUri,
+    },
+    target: {
+      funcName: options.targetFunc,
+    },
+  };
+  if (options.errorHandlerFunc) {
+    triggerOptions.errorHandler = {
+      funcName: options.errorHandlerFunc,
+    };
+  }
+  const result = await triggers.upsert(triggerName, triggerOptions);
+  logger.info(options.json ? JSON.stringify(result) : result.status);
+}
+
+async function triggerRemoveHandler(options) {
+  // options = {"_":["triggers","remove","triggerName"],"$0":"bn"}
+  // We have to use the position number here because of lack of good support
+  // for nested commands with positional arguments
+  // consider replacing with `nested-yargs` npm package
+  const triggerName = options._[2];
+  const result = await triggers.remove(triggerName);
+  logger.info(options.json ? JSON.stringify(result) : result.status);
+}
+
+async function triggerListHandler(options) {
+  const result = await triggers.list();
+  const rawData = (result.list || []).map(({ name, config, updatedAt }) => ({
+    name,
+    target: config.target.funcName,
+    errorHandler: config.errorHandler ? config.errorHandler.funcName : undefined,
+    lastUpdate: updatedAt,
+  }));
+  if (options.json) {
+    logger.info(JSON.stringify(rawData));
+  } else if (rawData.length > 0) {
+    logger.info(columnify(rawData, {
+      columns: ['name', 'target', 'errorHandler', 'lastUpdate'],
+      config: {
+        name: {
+          headingTransform: () => 'TRIGGER',
+          minWidth: 25,
+        },
+        target: {
```

Comment:
What about showing `source`? password is already redacted by server

Reply:
sure lets add it, but lets decide exactly what will be displayed
only `host` and `streamName`?
```
TRIGGER              SOURCE HOST     		SOURCE STREAM   TARGET               ERROR HANDLER        LAST UPDATE
test2                misc-redis.binaris.io  number		    hello                                     2019-03-21T09:38:55.875Z
```

Reply:
You'd need proto too, we'd like to support other triggers except for redis.
I'd have one column for `source` and create a formatter for each source.
For redis://:secret@host:666/0#streamName` you could print: `redis://host:666/0#streamName`
If port is the default redis port (6379 you could omit it).

Reply:
so maybe this should also come from the server? (as he is the one who does the parsing, he should know how to build the uri)

Reply:
Fixed so not output looks like that
```
TRIGGER         SOURCE                                           TARGET               LAST UPDATE
test2           redis+tls://misc-redis.binaris.io:6379/0#numbers hello                2019-03-21T09:38:55.875Z
```

Also removed the error handler as it's not implemented yet, left it as is when using `--json` flag